### DISCUSSION TITLE
Popup Summary View Idea

### DISCUSSION THREAD

**Discussion by bhamtil-twilio** [twilio-labs/paste#2544] [Ideas] [answered]
Hi team! I'm looking at a use case where I'd like to surface key information about something in a popover-like experience. It is meant to offer meaningful information in fewer clicks/page loads, especially in scenarios where the user may be looking at many items. For example, if I were looking at a list of errors in the Error Logs, could we surface information about that error while still on the list view page without having to load a separate page with error details? I'm envisioning something like a "quick view" interaction that is common on e-commerce sites when you hover over an item, or perhaps the panel that appears in JIRA when you click an issue from the Backlog to view more details. Have you considered an interaction like this before? Let me know if I can provide more context on where this idea came from. Would love to brainstorm solutions!

**Comment by shleewhite** [comment, marked as answer]
Hi @bhamtil-twilio,

This is a great idea and could potentially be a pattern! If you want, we could brainstorm on this in office hours.

As a starting point, our current Popover allows for lots of different kinds of content inside of it, and for control over placement and width. We will also be publishing the side variant of Modal in 2 releases.

One thing to note, if you show the panel on hover there needs to be a way to open the panel with a keyboard as well. For your example, I’m assuming the error name is a link that takes you to a detail page. So there would need to be a button next to the link that you can click to open the hover panel.
